iambic/plugins/v0_1_0/aws/organizations/scp/models.py

```diff
@@ -0,0 +1,582 @@
+from __future__ import annotations
+
+import asyncio
+from enum import Enum
+from itertools import chain
+from typing import TYPE_CHECKING, Callable, List, Optional, TypedDict, Union
+
+from pydantic import BaseModel as PydanticBaseModel
+from pydantic import Field, constr, validator
+
+from iambic.core.context import ctx
+from iambic.core.iambic_enum import Command
+from iambic.core.logger import log
+from iambic.core.models import (
+    AccountChangeDetails,
+    BaseModel,
+    ExpiryModel,
+    ProposedChange,
+    ProposedChangeType,
+)
+from iambic.core.utils import normalize_dict_keys, plugin_apply_wrapper
+from iambic.plugins.v0_1_0.aws.models import (
+    AccessModel,
+    AWSAccount,
+    AWSTemplate,
+    Description,
+    Tag,
+)
+from iambic.plugins.v0_1_0.aws.organizations.scp.utils import (
+    apply_update_policy,
+    apply_update_policy_tags,
+    apply_update_policy_targets,
+    create_policy,
+    delete_policy,
+    get_policy,
+    service_control_policy_is_enabled,
+)
+
+if TYPE_CHECKING:
+    from iambic.plugins.v0_1_0.aws.iam.models import Path
+    from iambic.plugins.v0_1_0.aws.iambic_plugin import AWSConfig
+    from iambic.plugins.v0_1_0.aws.models import AWSOrganization
+
+
+AWS_SCP_POLICY_TEMPLATE = "NOQ::AWS::Organizations::SCP"
+
+
+class StatementEffect(str, Enum):
+    ALLOW = "Allow"
+    DENY = "Deny"
+
+
+class ServiceControlPolicyTargetItemType(str, Enum):
+    ACCOUNT = "ACCOUNT"
+    ORGANIZATIONAL_UNIT = "ORGANIZATIONAL_UNIT"
+    ROOT = "ROOT"
+
+
+class ServiceControlPolicyItemType(str, Enum):
```

Comment:
Does the class name make sense here? OrganizationsPolicyType maybe does, but today we're only going to worry about Service Control Policies right? What's the need for mentioning ai services, backup, and tag policies?

Reply:
I added just to map the boto3's models. In list-policies and create-policies is being used. 
Renamed and added a comment.